Web Development/README.md

```diff
@@ -1312,6 +1312,11 @@ to everyone - everywhere!</td>
         <td>Youtube channel that teaches Web Development in a super simple way.Explore the content rich channel to become a pro in web development.
         </td>
       </tr>
+      <tr>
+        <td><a href="https://youtube.com/playlist?list=PLfqMhTWNBTe3H6c9OGXb5_6wcc1Mca52n&si=UYYv7L3NHDF31bHL">Apna College</a></td>
```

Comment:
```suggestion
        <td><a href="https://www.youtube.com/@ApnaCollegeOfficial">Apna College</a></td>
```

Reply:
Another change you need to make was the URL, change it.

Reply:
Ok sir will do it right a way.

Reply:
Done sir in #352